ioctl/cmd/did/diddoc.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package did
+
+const (
+	// DIDPrefix is the prefix string
+	DIDPrefix = "did:io:"
+	// DIDAuthType is the authentication type
+	DIDAuthType = "Secp256k1VerificationKey2018"
```

Comment:
from https://www.w3.org/ns/did/v1
it should be: EcdsaSecp256k1VerificationKey2019?

Reply:
> from https://www.w3.org/ns/did/v1
> it should be: EcdsaSecp256k1VerificationKey2019?
I am not sure,because https://www.w3.org/TR/did-core/ only show Secp256k1VerificationKey2018,I didn't find any more definitions.
In here,https://uniresolver.io/,did:ethr:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6 also used Secp256k1VerificationKey2018

Reply:
@xinxin-crypto can you take a look?

Reply:
The specification has been constantly updated during past years. It looks like we should use EcdsaSecp256k1VerificationKey2019 according to the lates spec at https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/.

Reply:
ok